pom.xml

```diff
@@ -1145,6 +1150,70 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <!--
+        the AWS module pulls in jackson; its transitive dependencies can create
+        intra-jackson-module version problems.
+        -->
+      <dependency>
```

Comment:
Continuing https://github.com/apache/spark/pull/12004#discussion_r113700119 - I think it's useful to put things in the parent when they're used across multiple modules. For something only referenced in one child POM, it is probably less overhead to just put it the dependency in the child rather than split and repeat part of it.